x-pack/plugins/cases/server/client/attachments/client.ts

```diff
@@ -33,6 +34,10 @@ export interface AttachmentsSubClient {
    * Adds an attachment to a case.
    */
   add(params: AddArgs): Promise<ICaseResponse>;
+  /**
+   * Retrieves the number of alerts attached to a case.
+   */
+  countAlertsAttachedToCase(caseId: string): Promise<AlertsCount>;
```

Comment:
Just a side note, this does not have an equivalent API route. That should be fine though since it's exposed through the metrics route.